SCons/Defaults.py

```diff
@@ -542,43 +572,69 @@ def __call__(self, *args, **kw):
             frame = frame.f_back
         return None
 
-# if $version_var is not empty, returns env[flags_var], otherwise returns None
+
 def __libversionflags(env, version_var, flags_var):
+    """
+    if $version_var is not empty, returns env[flags_var], otherwise returns None
```

Comment:
just asking - is None a suitable return value here (I realize it already was that) - applies to new bit below also.